sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/impl/JdbcDriverTest.java

```diff
@@ -198,6 +205,91 @@ public void testSelectsFromExistingTable() throws Exception {
     assertThat(resultRows, containsInAnyOrder(row(1L, "aaa"), row(2L, "bbb")));
   }
 
+  @Test
+  @Ignore("https://issues.apache.org/jira/browse/CALCITE-2394")
+  public void testTimestampWithDefaultTimezone() throws Exception {
+    TestTableProvider tableProvider = new TestTableProvider();
+    Connection connection = JdbcDriver.connect(tableProvider);
+
+    // A table with one TIMESTAMP column
+    Schema schema = Schema.builder().addDateTimeField("ts").build();
+    connection.createStatement().executeUpdate("CREATE TABLE test (ts TIMESTAMP) TYPE 'test'");
+
+    ReadableInstant july1 =
+        ISODateTimeFormat.dateTimeParser().parseDateTime("2018-07-01T01:02:03Z");
```

Comment:
This is taking an opinion that no timezone offset is the same as a zero offset. This is definitely a opinion but probably not one everyone will agree with. (No offset being the local timezone is also a common opinion, one that will probably make this test pass.)